pkg/cmd/service/service_test.go

```diff
@@ -562,6 +562,9 @@ func TestHTTPServingTLS(t *testing.T) {
 
 		ensureServiceUp(t, nil)
 
+		_, err = retryablehttp.NewClient().Get("http://localhost:8080/healthz")
```

Comment:
This should be happening within `ensureServiceUp` otherwise you'd be copy pasting everywhere.

Reply:
Moved to `ensureServiceUp`. Thanks!